### PR TITLE
Update docs for metrics powerstore

### PR DESCRIPTION
**Pull request by tdawe**
# Description

Updated Observability documentation with references to PowerStore:
- Added description and link to the GitHub repository from the main README.md
- Updated Karavi Observability Capabilities with details about Volume metrics being supported for PowerStore
- Updated Supported Platforms with reference to the PowerStore versions that are supported (reference: https://dell.github.io/storage-plugin-docs/docs/dell-csi-driver/#supported-platforms)
- Added details about updating the PowerStore system credentials

# Issues

List the issues impacted by this PR:

| Issue ID |
| -------- |
|  https://github.com/dell/karavi-observability/issues/48        |

# Checklist:

- [x] I have performed a self-review of my own changes.

### CODE REVIEW

**Comment by ioeicicdsvc** [comment]
@tdawe: Thank you for submitting your pull request!

Please make sure you have followed our contributing guidelines. We will review it as soon as possible.

------------------------------------------------------------------------------------------------------------------------
We want your feedback! If you have any questions or suggestions regarding our contributing process/workflow, please reach out to us at karavi@dell.com.

**Review by rsedlock1958 — COMMENTED** [review]
Looks good. I do not have any editorial revisions.

File: README.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ Several services compose Karavi Observability, each living in their own GitHub r
...
+| Performance Metrics for PowerStore | [Karavi Metrics for PowerStore](https://github.com/dell/csm-metrics-powerstore) | Performance Metrics for PowerStore captures telemetry data about Kubernetes storage usage and performance obtained through the CSI (Container Storage Interface) Driver for Dell EMC PowerStore. The metrics service pushes it to the OpenTelemetry Collector, so it can be processed, and exported in a format consumable by Prometheus. Prometheus can then be configured to scrape the OpenTelemetry Collector exporter endpoint to provide metrics so they can be visualized in Grafana. Please visit the repository for more information. |
```
Comment (gallacher) [inline README.md:24]: Can we annotate these sections indicating the release they will be available?

Comment (tdawe) [reply README.md]: Done

File: docs/GETTING_STARTED_GUIDE.md [hunk 2]
```diff
@@ -14,22 +14,22 @@ This project provides Kubernetes administrators insight into CSI (Container Stor
...
-| -------- | --------- |
-| Storage Pool Consumption By CSI Driver | Yes |
-| Storage System I/O Performance By Kubernetes Node | Yes |
-| CSI Driver Provisioned Volume I/O Performance | Yes |
-| CSI Driver Provisioned Volume Topology | Yes |
+| Feature | PowerFlex | PowerStore |
+| -------- | --------- | --------- |
+| Storage Pool Consumption By CSI Driver | Yes | No |
+| Storage System I/O Performance By Kubernetes Node | Yes | No |
+| CSI Driver Provisioned Volume I/O Performance | Yes | Yes |
```
Comment (gallacher) [inline docs/GETTING_STARTED_GUIDE.md:21]: Annotate this as well "As of Release 0.4.0"

File: docs/GETTING_STARTED_GUIDE.md [hunk 3]
```diff
@@ -14,22 +14,22 @@ This project provides Kubernetes administrators insight into CSI (Container Stor
...
 ## Supported Platforms
 
 The following matrix provides a list of all supported versions for each Dell EMC Storage product.
 
-| Platforms | PowerFlex |
-| -------- | --------- |
-| Storage Array | v3.0, v3.5 |
-| Kubernetes | 1.18, 1.19, 1.20 |
-| Openshift | 4.5, 4.6 |
+| Platforms | PowerFlex | PowerStore |
```
Comment (gallacher) [inline docs/GETTING_STARTED_GUIDE.md:28]: Perhaps annotate the "PowerStore" column heading: As of Release 0.4.0

File: docs/GETTING_STARTED_GUIDE.md [hunk 4]
```diff
@@ -145,6 +145,18 @@ $ kubectl delete secret vxflexos-config -n karavi
 $ kubectl get secret vxflexos-config -n vxflexos -o yaml | sed 's/namespace: vxflexos/namespace: karavi/' | kubectl create -f -
 ```
 
+### PowerStore
```
Comment (gallacher) [inline docs/GETTING_STARTED_GUIDE.md:148]: Annotate As of Release 0.4.0

File: docs/GETTING_STARTED_GUIDE.md (hunk 2 above)
Comment (tdawe) [reply docs/GETTING_STARTED_GUIDE.md:21]: Done

File: docs/GETTING_STARTED_GUIDE.md (hunk 3 above)
Comment (tdawe) [reply docs/GETTING_STARTED_GUIDE.md]: Done

File: docs/GETTING_STARTED_GUIDE.md (hunk 4 above)
Comment (tdawe) [reply docs/GETTING_STARTED_GUIDE.md]: Done

**Review by LJ-Software — APPROVED** [review]

**Review by ameer-jabbar — APPROVED** [review]

**Pull request merged**